tests/func/test_repo_index.py

```diff
@@ -200,7 +200,7 @@ def test_dumpd(dvc):
 
 
 def test_unique_identifier(tmp_dir, dvc, scm, run_copy):
-    dvc.config["core"]["autostage"] = True
+    dvc.scm_context.autostage = True
```

Comment:
Changing config won't really work as `SCMContext` determines `autostage` config when it's initialized.